packages/e2e/tests/storedCard/general.test.js

```diff
@@ -6,7 +6,7 @@ const cardPage = new CardComponentPage('.stored-card-field', {}, 'storedcards');
 
 const EMPTY_FIELD = LANG['error.va.sf-cc-cvc.01'];
 
-fixture`Testing some general functionality and UI on the stored card component`.beforeEach(async t => {
+fixture.only`Testing some general functionality and UI on the stored card component`.beforeEach(async t => {
```

Comment:
Is it intended to leave the `.only` ?

Reply:
No! I searched for that...
...aah! but my filter excluded .test files! ;-(